package.json

```diff
@@ -15,12 +15,17 @@
   "dependencies": {
     "bootstrap": "^4.3.1",
     "classnames": "^2.2.6",
+    "connected-react-router": "^6.3.2",
     "formik": "^1.5.1",
+    "history": "^4.9.0",
     "ramda": "^0.26.1",
     "ramda-adjunct": "^2.16.1",
     "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-redux": "^6.0.1",
+    "react-router": "^5.0.0",
+    "react-router-bootstrap": "^0.25.0",
```

Comment:
Hi, let me share small snippet then you probably wont needed this `react-router-bootstrap`

```js
import { NavLink } from 'reactstrap';
import { NavLink as Link } from 'react-router-dom';
///....
<NavLink tag={Link} to={`/${urls.catalog}`}>
```